lib/agent0/agent0/hyperdrive/policies/lpandarb.py

```diff
@@ -92,7 +94,7 @@ def calc_reserves_to_hit_target_rate(
     total_shares_needed = FixedPoint(0)
     total_bonds_needed = FixedPoint(0)
     # pylint: disable=logging-fstring-interpolation
-    logging.debug(f"Targeting {float(target_rate):.2%} from {float(interface.calc_fixed_rate()):.2%}")
+    logging.warning(f"Targeting {float(target_rate):.2%} from {float(interface.calc_fixed_rate()):.2%}")
```

Comment:
I don't think this is a warning, as there is nothing wrong when it is posted. If it's not helpful in your debug then maybe we should just delete it all together? Lots of non-helpful logs can be worse than few logs.

Reply:
I changed this because running pytest with log level of debug returns like 10,000 lines